src/main/java/com/mentalhealthapi/mentalhealthapi/MentalHealthController.java

```diff
@@ -56,21 +60,20 @@ public ResponseEntity fetchBlogs(@PathVariable("name") String name) {
      */
     @PostMapping(value = "/disorders/{name}/blog", consumes = "application/json", produces = "application/json")
     public Blog createBlog(@PathVariable("name") String name, @RequestBody Blog blog) {
-        // we're going to need a user id passed here as well. for now, let's not worry about it
-
-        // for not, just return created blog as a stub
+        // TODO(Spurlock): Pass userId.
+        // TODO(Spurlock): Make Blog return more than a stub.
```

Comment:
Once again just fixing comments to be more consistent and clear. 

https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3